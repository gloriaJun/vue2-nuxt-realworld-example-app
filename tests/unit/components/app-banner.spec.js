import { shallowMount } from '@vue/test-utils';
import AppBanner from '@/components/app-banner.vue';

const siteName = 'siteName';

describe('Banner.vue', () => {

  it('should match the snapshot', () => {
    const wrapper = shallowMount(AppBanner, {
      propsData: { siteName: siteName },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

})
