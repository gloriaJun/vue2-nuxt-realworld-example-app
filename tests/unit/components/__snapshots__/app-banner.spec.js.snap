// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner.vue should match the snapshot 1`] = `
<div class="banner">
  <div class="container">
    <h1 class="logo-font">
      siteName
    </h1>
    <p>A place to share your knowledge.</p>
  </div>
</div>
`;
